<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>241</width>
      <height>693</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Starting Net (config or trained)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLineEdit" name="txtNetConfig">
         <property name="text">
          <string>/Users/connorbowley/Dropbox/SHARE/neuralNetworks/NetConfigs/nc_bn18x18x3-3.txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNetConfig">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Training Data</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="txtTrainingData">
         <property name="text">
          <string>/Users/connorbowley/Development/neuralNetwork/Full_loop/Idxs/final_expert.idx</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTrainingData">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Training Labels</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="txtTrainingLabels">
         <property name="text">
          <string>/Users/connorbowley/Development/neuralNetwork/Full_loop/Idxs/final_expert_species.idx</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTrainingLabel">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output CNN Base Name</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="txtCNNName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Training Mosaic Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="txtTrainMosaic">
         <property name="text">
          <string>/Users/connorbowley/Development/neuralNetwork/mosaic_test_images/Thesis/train</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTrainMosaic">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Test Mosaic Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="txtTestMosaic">
         <property name="text">
          <string>/Users/connorbowley/Development/neuralNetwork/mosaic_test_images/Thesis/test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTestMosaic">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>MSI Locations Binary</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="txtMSILocations">
         <property name="text">
          <string>/Users/connorbowley/Development/neuralNetwork/rsrc/old/msi_locations.bin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMSILocations">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnIterations">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Output Location</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="txtOutputLocation">
         <property name="text">
          <string>./</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOutputLocation">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>neuralNetwork Build Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="txtBuildDir">
         <property name="text">
          <string>./</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBuildDir">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="txtMessageBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
